### DISCUSSION TITLE
Help Getting Started With VR For My Company

### DISCUSSION THREAD

**Discussion by hudlander** [community/community#39279] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I work for a government consulting firm and we are trying to evaluate if it makes sense to invest in VR.  I myself have no programming or tech background and am kind of lost.  Specifically, I need:

1) To understand how to actually build VR solutions and ultimately evaluate if it makes sense for us to build them in-house. 
-As in, exactly what is required to do this—technology, capabilities, etc.  

We are looking at multiple programs, but for the purposes right now, let's say how to build VR to consult with Police departments on bias training/active shooter, just as an example.

Any help would be much appreciated, I am lost!  Thanks.

**Comment by queenofcorgis** [comment]
Hi @hudlander,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on VR. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by hudlander** [reply]
Thanks, where would that be?  I could not figure it out.
